tests/unit/transformers/test_utils.py

```diff
@@ -550,17 +550,20 @@ def test__safe_parse_datetime():
     """
     # Setup
     str_input = '2023-01-01 12:00:00+0200'
+    str_input_no_format = '20220902110443000000'
```

Comment:
what is the case where an input string like this would come up? It doesn't seem to be a valid datetime since I think pandas only accepts up to microseconds and this goes 10^-8